src/search/search-index/add.js

```diff
@@ -184,3 +225,31 @@ async function performIndexing(indexDoc) {
         console.error(err)
     }
 }
+
+async function addMetaToReversePage(type, pageId, timestamp) {
```

Comment:
Same here - in need of some JSDoc. Even just the `@param` and `@returns` types is fine as this fn will never be used externally.